src/guess-number.app.js

```diff
@@ -0,0 +1,60 @@
+// What do I need to import from other js files?
+import compareNumbers from './guess-number.js';
+
+
```

Comment:
I really like the way you outlined out what you needed to do with comments. In the future though, please delete these comments before you submit your code!